source/developer/bot-accounts.rst

```diff
@@ -146,22 +147,22 @@ In addition to email, you may identify the user by its username or user ID.
 Bot accounts which were converted from user accounts will have their authentication data cleared if they were email/password accounts. Those synced from LDAP/SAML will not have their authentication data cleared so that LDAP/SAML sync performs correctly.
```

Comment:
```suggestion
Bot accounts which were converted from user accounts will have their authentication data cleared if they were email/password accounts. Those synchronized from LDAP/SAML will not have their authentication data cleared so that LDAP/SAML synchronization performs correctly.
```